app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -112,6 +133,7 @@ public void NavigationTest() {
         try{
             Thread.sleep(500);
         }catch(Exception e){}
+        intended(hasComponent(UploadActivityActivity.class.getName()));
```

Comment:
I wonder why you don't have the null instance error, when loading the navigation with a null user. That was a big issue for me when I tried making the navigation tests

Reply:
As discussed on Monday, I used the same technique as for the navigation. Basically, there is a variable to choose if the header will be displayed or not (and thus if the user's info should be retrieved). It may be a bit dirty but it worked for me even if I should look more closely because some behavior seems strange.